content/tutorial/02-sveltekit/02-routing/01-pages/README.md

```diff
@@ -4,13 +4,28 @@ title: Pages
 
 SvelteKit uses filesystem-based routing, which means that the _routes_ of your app — in other words, what the app should do when a user navigates to a particular URL — are defined by the directories in your codebase.
 
-In this app we have two routes — `src/routes/+page.svelte`, which maps to `/`, and `src/routes/about/+page.svelte`, which maps to `/about`. Clicking the `about` link will take you from the home page to the about page.
+The routes are located within `src/routes`. Every directory within which contains a `+page.svelte` file creates a route in your app.
 
-> Unlike traditional multi-page apps, navigating to `/about` updates the contents of the current page, like a single-page app. This gives us the best of both worlds — fast server-rendered startup, then instant navigation.
+In this app we currently have one route — `src/routes/+page.svelte`, which maps to `/`.
 
-Let's add a link in `src/routes/about/+page.svelte` back to the homepage:
+Let's add a second route, `src/routes/about/+page.svelte`, which maps to `/about`:
```

Comment:
What do you mean? You should be able to left click and a context menu appears so you can add those files/folders

Reply:
nope. left click where?

relatedly, i did wonder if a constrained way to allow the user to create new files would be if the auto-generated links like in the left-hand column like this one...

<img width="658" alt="image" src="https://user-images.githubusercontent.com/1162160/205352457-51a46484-71cb-4b13-a2c5-df6599a7ef48.png">

...created the file if it didn't already exist. maybe that's a confusing UI, but i figured I'd throw it out there

Reply:
Oh woops I meant right click, in the file tree on one of the folders or files

Reply:
Ah okay it works on folders — I could have sworn I tried that. I think the context menu is a little undiscoverable — what if icons appeared in the right margin on hover?

Reply:
It's restricted per tutorial - you can only add/delete the files/folders you need for the tutorial to complete, that's why in that case you don't get it for files, because there's no need to rename/delete a file in that chapter. Making it more discoverable: I was expecting it to feel natural because that's what you do in the regular explorer view on your desktop, too. But it's probably not in the tutorial if you start with Svelte tutorial chapters where there's no possibility to add/remove anything.

Reply:
I had no trouble with the file tree. My one suggestion for it was going to be if we could add vertical lines showing which folder a file/folder was in (VS Code does this. you also do it right below when you type it out with `├` and `|` characters)